components/event/EditForm.vue

```diff
@@ -201,10 +201,16 @@
     await $fetch('/api/event', {
       method: 'PUT',
       body: state,
-    }).catch((error) => displayErrorAlertFromMessage(error.statusMessage))
-
-    displaySuccessAlert('alert.success.event.edit.modified')
-    setTimeout(() => navigateTo(localePath('/admin/events')), 2000)
+    })
+      .then((_data) => {
+        // Handle successful response
+        displaySuccessAlert('alert.success.event.edit.modified')
+        setTimeout(() => navigateTo(localePath('/admin/events')), 2000)
+      })
+      .catch((error) => {
+        // Handle errors, including HTTP errors
```

Comment:
Maybe add a custom error message for this type of error, as is isn't easy to spot when adding datetime as ISO8601.